specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/privateLinkService.json

```diff
@@ -422,6 +422,90 @@
           "final-state-via": "location"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/locations/{location}/checkPrivateLinkServiceVisibility": {
+      "post": {
+        "tags": [
+          "PrivateLinkServices"
+        ],
+        "operationId": "CheckPrivateLinkServiceVisibility",
+        "description": "Checks the subscription is visible to private link service",
+        "parameters": [
+          {
+            "name": "location",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The location of the domain name."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. Returns whether the subscription is visible to private link service.",
+            "schema": {
+              "$ref": "#/definitions/PrivateLinkServiceVisibility"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Check private link service visibility": {
+            "$ref": "./examples/CheckPrivateLinkServiceVisibility.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/locations/{location}/autoApprovedPrivateLinkServices": {
+      "get": {
+        "tags": [
+          "PrivateLinkServices"
+        ],
+        "operationId": "AutoApprovedPrivateLinkServices_List",
```

Comment:
Since this method is related to privateLinkService, maybe it would be better to name it `PrivateLinkServices_ListAutoApprovedPrivateLinkServices`

Reply:
Fixed.